cost-analyzer/templates/aggregator-statefulset.yaml

```diff
@@ -67,6 +67,9 @@ spec:
     {{- end }}
       serviceAccountName: {{ template "aggregator.serviceAccountName" . }}
       volumes:
+        - name: aggregator-staging
+          emptyDir:
+            sizeLimit: {{ .Values.kubecostAggregator.stagingEmptyDirSizeLimit }}
```

Comment:
Should you set a default value here in case, somehow, a user unsets the corresponding value? I presume this will fail otherwise but not sure how big of a potential problem that would be considering the StatefulSet option is probably not the majority use case anyhow.

Reply:
How common is it for someone to unset a value in `values.yaml`? I dislike the multi-default convention (one in values.yaml, one in the template) but if its a common occurrence I can shift.

Reply:
Uncommon. I vote go with as-is since I agree that values file should be authoritative.